### ISSUE TITLE
SQL frontend

### ISSUE THREAD

**Issue by psubotic** [souffle-lang/souffle#200]
For the purpose of benchmarks and perhaps a use case would be nice to have an SQL frontend. This could be a SQL to Datalog translator or even better SQL to RAM frontend.

**Comment by b-scholz** [comment]
This is an interesting one. The idea would be to get a lot of SQL benchmarks translated to Datalog. Most of these benchmarks will have  a non-recursive nature. Hence, we would get a benchmark suite which would stress-test the join/scheduling performance of Soufflé. 

In addition, we do not have full support of standard data-types (floating-point, blobs, etc.). We would need to implement most of these data-types for attributes first before we can think about translating SQL benchmarks to Datalog.

**Comment by scyptnex** [comment]
Note that SQL comes in dialects.
Mysql, postgres, oracleDB, Microsoft etc

[Going by the w3c](http://www.w3schools.com/sql/sql_quickref.asp) it would be reasonably easy for most things, though we'd have to decide things like "does create view create a new IDB or does it act like an alias to an existing IDB?"

**Comment by psubotic** [comment]
yes I'd assume most cross database benchmarks conform to some SQL standard

**Comment by b-scholz** [comment]
Perhaps we defer this issue and close it.
